_pages/webextensions.md

```diff
@@ -12,160 +12,173 @@ parent: activities
+| --------------------- |:-------------------------------------:|:-----------------------------:|
+| [Content scripts](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Content_scripts)               | [Borderify](https://github.com/mdn/webextensions-examples/tree/master/borderify)                    | [Content Script Tutorial](https://youtu.be/WrEgBDg56sM)       |
+| [browserAction API](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/browserAction)      | [Bookmark It](https://github.com/mdn/webextensions-examples/tree/master/bookmark-it)                              |   |
+| [pageAction API](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/pageAction)    | [Chill Out](https://github.com/mdn/webextensions-examples/tree/master/chill-out)                    | [pageAction tutorial ](https://www.youtube.com/watch?v=RBI-j8USuJs&feature=youtu.be)                      |
+|                       | [Apply CSS](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/pageAction)                                  |                               |
+|                       | [IP-Finder](https://github.com/tsl143/jsDemos/tree/master/ipFinder_webext)                                  |                               |
+|                       | [Inshorts](https://github.com/viswanathamsantosh/inshorts/tree/page-action)                                  |                               |
 
-__Duration:__ For duration, we recommend 1 - 3 hours with 5+ attendees. However you may choose to run the workshop as a series over a shorter time but multiple days.
 
-Attendees should bring their own laptops to the workshop, or the workshop should be held in a computer lab.
 
-You can make a copy and customize the [Build Your Own Extension Add-on template presentation](https://docs.google.com/presentation/d/1n7FEBCxSJdbfui2qjJv8vptsuchLB8-KES3SULVMI6w/edit#slide=id.g1bf30aaa6e_0_45) for your workshop.
 
-There are two options for facilitation:
-1. Self-facilitate without a mentor.
-  * If questions arise or if help is needed during the event, feel free to join the IRC channels #extdev and #webextensions to ask the community for support.
+* 2nd half of the meeting: ask attendees to put these APIs to use by creating an original extension or by building one of the examples listed in the table above.  
 
-2. Ask a person with experience building browser extensions to provide guidance to attendees.
-  * Need help finding an experienced mentor? Send an email to cneiman \[at\] mozilla \[dot\] com with the link to your Reps event page and anticipated number of attendees.
-  * Depending on location and availability, Mozilla might be able to send a staff member or volunteer expert to co-facilitate.
+💡 If your attendees need help coming up with an idea for building an original extension, ask them to visit the[ repository of extension ideas](https://github.com/firefox-addons/ideas/issues) for inspiration. 
```

Comment:
can you please add a space before the [ ? Otherwise the space is included in the link description.